integration-tests/README.md

```diff
@@ -0,0 +1,42 @@
+# Integration Tests
+
```

Comment:
Can add prerequisite of solc v0.8.0

I'm getting this error
```
$  ./run.sh 
...
thread 'main' panicked at 'solc compile error: Io(Os { code: 2, kind: NotFound, message: "No such file or directory" })', integration-tests/src/bin/gen_blockchain_data.rs:51:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Reply:
I've added a `Requirements` section in the readme :)